src/sentry/auth/elevated_mode.py

```diff
@@ -1,38 +1,47 @@
 from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Tuple
+
+
+class InactiveReason(str, Enum):
+    INVALID_IP = "invalid-ip"
+    INCOMPLETE_SSO = "incomplete-sso"
```

Comment:
Should the string values be consistent with `_` as well, or is `-` the pattern we have to stick to?

Reply:
Answered this myself, this is ported over from the existing values as is, so might be needed with the hyphen/dash